lib/sdam/topology.js

```diff
@@ -698,6 +734,7 @@ class Topology extends EventEmitter {
    * @param {boolean} [options.ignoreUndefined=false] Specify if the BSON serializer should ignore undefined fields.
    * @param {ClientSession} [options.session=null] Session to use for the operation
    * @param {object} [options.topology] The internal topology of the created cursor
+   * @param {{new(topology: any, ns: string | MongoDBNamespace, cmd: object, options: object)}} [options.cursorFactory] A Cursor constructor
```

Comment:
this looks like it needs a typedef, I think you might be able to just put `CoreCursor` here